task5.js

```diff
@@ -0,0 +1,22 @@
+function range(start, end, step) {
+    if (step === undefined) {
+        step = 1;
+    }
+    let range = new Array(Math.round((Math.abs(start - end) + 1) / step)), sum = 0;
+    if (start > end) {
+        step = -step;
+    }
+    for (let i = 0; i < range.length; i++, start += step) {
```

Comment:
Think of `i = start` to short code a little.

Reply:
yes

Reply:
sure
stupid of me to ask

Reply:
it's all okay